pinot-common/src/main/java/org/apache/pinot/common/segment/generation/SegmentGenerationUtils.java

```diff
@@ -87,7 +94,12 @@ public static Schema getSchema(String schemaURIString) {
     }
   }
 
+  @Deprecated
```

Comment:
users can keep *configuring* nulls, but any dev work should consider authentication from this point on. otherwise, we end up with a nightmare of a compatibility matrix for new features and auth support

Reply:
Sounds good.